.editorconfig

```
@@ -17,3 +17,7 @@ dotnet_sort_system_directives_first = true
 
 dotnet_diagnostic.IDE0073.severity = warning  # Require file header
 file_header_template = Copyright (c) Duende Software. All rights reserved.\nSee LICENSE in the project root for license information.
+
+# Require file scoped namespaces
+csharp_style_namespace_declarations = file_scoped:error
+dotnet_diagnostic.IDE0161.severity = error
```

This is the convention we are using now (see rest of file):

- Comment from the [diagnostic title](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0160-ide0161#options)
- Options grouped under the related rule

(Also missing a final new line.)

```suggestion

dotnet_diagnostic.IDE0161.severity = error  # Namespace declaration preferences
csharp_style_namespace_declarations = file_scoped:error

```